src/Cello/test_ViewTestRoutines.hpp

```diff
@@ -0,0 +1,183 @@
+// See LICENSE_CELLO file for license and copyright information
+
+/// @file     test_ViewTestRoutines.cpp
+/// @author   Matthew Abruzzo (matthewabruzzo@gmail.com)
+/// @date     2023-07-03
+/// @brief    Common routines used for testing CelloView and related machinery
+
+#ifndef TEST_VIEW_TEST_ROUNTINES_HPP
```

Comment:
I'm not entirely sure what a "ROUNTINE" is... I think that this is a typo (same note for the line immediately below)

Reply:
Thanks for catching that!